auth-web/src/util/constants.ts

```diff
@@ -122,10 +122,23 @@ export enum LoginSource {
     BCEID = 'BCEID'
 }
 
-export enum TransactionStatus {
-    COMPLETED = 'Completed',
-    PENDING = 'Pending',
-    CANCELLED = 'Cancelled'
+// keep in sync with the pay-api invoice status enum
+export enum InvoiceStatus {
+    APPROVED = 'APPROVED',
+    CANCELLED = 'CANCELLED',
+    CREATED = 'CREATED',
+    CREDITED = 'CREDITED',
+    COMPLETED = 'COMPLETED', // NOTE: this === PAID value (api alters it from PAID to COMPLETED in postdump)
```

Comment:
It would be nice to change this back to PAID  in the API

Reply:
Yeah -- I've made it so that the UI will be fine either way. I think entities/ppr/others may need to update code before we make that change